atlas-search/public-api/index.md

```diff
@@ -2,21 +2,28 @@
 
 ## Overview
 
-The index API includes two mutations for creating or updating a document and deleting a document.
+The public API includes mutations:
 
-- The `index` mutation accepts a `DocumentInput` as a parameter and returns a `DocumentMutationResponse`.
-- The `delete` mutation accepts an `id` as a parameter and returns a `DocumentMutationResponse`.
+- [index](#creating-or-updating-a-document) mutation for creating or updating a document
+- [delete](#deleting-a-document) mutation for deleting a document
+- [deleteAll](#deleting-all-documents) mutation for deleting all documents
+
+To use a mutation you need to [authenticate](#authentication).
 
 ---
 
 ## Authentication
 
-Clients calling the index API are required to add an authentication header with the valid authentication token.
+Clients calling the API mutations are required to add an authentication header with the valid authentication token.
 
 ```
 Authorization: Bearer {ACCESS_TOKEN}
 ```
 
+Data needed to consume API you can find in your WordPress Atlas Search Plugin in admin area in section `Settings`.
```

Comment:
This could potentially be shortened to:
Data needed to consume API can be found in your WP Admin area in the Atlas Search tab under `Settings`.